Source/Core/Core/Boot/Boot.cpp

```diff
@@ -620,6 +626,11 @@ bool CBoot::BootUp(Core::System& system, const Core::CPUThreadGuard& guard,
 
     bool operator()(const BootParameters::DFF& dff) const
     {
+#ifdef USE_RETRO_ACHIEVEMENTS
+      AchievementManager::GetInstance()->HashGame(dff.dff_path,
+                                                  [](AchievementManager::ResponseType r_type) {});
```

Comment:
DFF files aren't games. You probably shouldn't initialize achievements at all for them.